zitify/zitify

```diff
@@ -19,4 +19,65 @@
 preload_dir=$(dirname ${BASH_SOURCE[0]})
 preload_lib=${preload_dir}/libzitify.so
 
+usage() {
+    echo "zitify <cmd> [args]"
+    echo "       Run zitified <cmd>"
+    echo ""
+    echo "zitify enroll -j <enrollment token> -i <identity file> [ -k|--key <private key> [ -c|--cert certificate ] ]"
+    echo "       Enroll identity"
+}
+
+function enroll() {
+  python - $* <<EOF
```

Comment:
I don't think we support macOS yet, but will we see systems that need to use `python3`?